src/main/java/com/wcc/platform/service/CmsService.java

```diff
@@ -77,8 +74,8 @@ public CollaboratorPage getCollaborator() {
    */
   public CodeOfConductPage getCodeOfConduct() {
     try {
-      final File file = Path.of(FileUtil.getFileUri(CODE_OF_CONDUCT.getFileName())).toFile();
-      return objectMapper.readValue(file, CodeOfConductPage.class);
+      final var data = FileUtil.readFileAsString(CODE_OF_CONDUCT.getFileName());
+      return objectMapper.readValue(data, CodeOfConductPage.class);
     } catch (IOException e) {
```

Comment:
Exception type needs to be changed to JsonProcessingEception in all the methods to be consistent